underscore.js

```diff
@@ -282,11 +291,21 @@
   };
 
   // Invoke a method (with arguments) on every item in a collection.
-  _.invoke = restArgs(function(obj, method, args) {
-    var isFunc = _.isFunction(method);
-    return _.map(obj, function(value) {
-      var func = isFunc ? method : value[method];
-      return func == null ? func : func.apply(value, args);
+  _.invoke = restArgs(function(obj, path, args) {
+    var contextPath = [], func;
+    if (_.isFunction(path)) {
+      func = path;
+    } else if (_.isArray(path)) {
+      contextPath = path.slice(0, -1);
+      path = path[path.length - 1];
+    }
+    return _.map(obj, function(context) {
+      if (!func) {
+        if (contextPath.length) context = deepGet(context, contextPath);
+        if (context == null) return void 0;
+        func = context[path];
```

Comment:
Thanks for catching this! Fixed https://github.com/jashkenas/underscore/pull/2620/commits/d3a1020d9d61b99d5dbe182ab240010dd6370375